command/agent.go

```diff
@@ -507,10 +511,14 @@ func (c *AgentCommand) Run(args []string) int {
 			return 1
 		}
 
-		if config.DisableIdleConnsAutoAuth {
+		if config.DisableIdleConnsCaching {
```

Comment:
Why did this change for `proxyClient`, but not for the other clients?

Reply:
Was actually a bug I found, this should have been the proxy client.

Reply:
:+1: